tests/integration-tests/src/fixtures/council/FailToElectCouncilFixture.ts

```diff
@@ -0,0 +1,214 @@
+import { BaseQueryNodeFixture, FixtureRunner } from '../../Fixture'
+import { AddStakingAccountsHappyCaseFixture, BuyMembershipHappyCaseFixture } from '../membership'
+import { blake2AsHex } from '@polkadot/util-crypto'
+import { MINIMUM_STAKING_ACCOUNT_BALANCE } from '../../consts'
+import { ForumThreadWithInitialPostFragment, ThreadCreatedEventFieldsFragment } from '../../graphql/generated/queries'
+import { assertCouncilMembersRuntimeQnMatch } from './common'
+import { Balance } from '@polkadot/types/interfaces'
+import { MemberId } from '@joystream/types/common'
+import { assert } from 'chai'
+
+interface IScenarioResources {
+  candidatesMemberIds: MemberId[]
+  candidatesStakingAccounts: string[]
+  candidatesMemberAccounts: string[]
+  councilCandidateStake: Balance
+  councilMemberIds: MemberId[]
+}
+
+export class FailToElectCouncilFixture extends BaseQueryNodeFixture {
+  /*
+    Execute all scenarios when a new council election is expected to fail.
+  */
+  public async execute(): Promise<void> {
```

Comment:
I think a fixture is supposed to be more atomic/reusable piece of code, this sounds more like a flow/scenario.

Reply:
The fixture has been split into two, and the flow calls them in a sequence.